### DISCUSSION TITLE
Documentation for trivy debian installation no longer works

### DISCUSSION THREAD

**Discussion by dannystaple** [aquasecurity/trivy#4904] [Bugs]
### Description

Since release v0.44.0, installation into debian/ubuntu using instructions at https://aquasecurity.github.io/trivy/v0.44/getting-started/installation/ (or previous instrucions) result in an error.

### Desired Behavior

The `sudo apt-get install trivy` command at the end of the process would install the trivy package.

### Actual Behavior

```
sudo apt-get install wget apt-transport-https gnupg lsb-release
wget -qO - https://aquasecurity.github.io/trivy-repo/deb/public.key | gpg --dearmor | sudo tee /usr/share/keyrings/trivy.gpg > /dev/null
echo "deb [signed-by=/usr/share/keyrings/trivy.gpg] https://aquasecurity.github.io/trivy-repo/deb $(lsb_release -sc) main" | sudo tee -a /etc/apt/sources.list.d/trivy.list
sudo apt-get update
sudo apt-get install trivy
```

results in:
```
# apt-get install trivy
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package trivy
```

### Reproduction Steps

```bash
1. Set up a debian/ubuntu environment which does not yet have trivy installed.
2. Follow the steps at https://aquasecurity.github.io/trivy/v0.44/getting-started/installation/ for debian/ubuntu.
3. It will fail to locate the package.
```

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
N/A - fails to install
```

### Operating System

debian or ubuntu

### Version

```bash
v0.44.0
```

### Checklist

- [ ] Run `trivy image --reset`
- [x] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by DmitriyLewen** [comment]
duplicate of #4908. 
PS. We updated `trivy-repo`. Install should work correctly.
